js/utils.js

```diff
@@ -17,6 +17,50 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
   };
   var computedFieldClashes = [];
 
+  function setLineClamps (selector) {
+    return setClampHeight(selector)
+      .then(function (elements) {
+        Superclamp.register(elements);
+        removeClampHeight(elements);
+      })
+      .catch(function (selector) {
+        console.error('Fail to clamp line in the selector ' + selector);
+      });
+  }
+
+  function setClampHeight (selector) {
+    selector = selector || '[data-line-clamp]';
+
+    return new Promise(function (resolve, reject) {
+      var $elements = $(selector);
+      
+      if (!$elements.length) {
+        reject(selector);
```

Comment:
I think this needs a `return` after this to stop the execution.

```suggestion
        reject(selector);
```